config/locales/en.yml

```diff
@@ -436,12 +436,9 @@ en:
       in_soft_max_range:
         message: "Net income is higher than expected based on the lead tenant’s working situation. Are you sure this is correct?"
     rent:
-      min:
-        title_text: "You told us the rent is %{brent}"
-        hint_text: "The minimum rent expected for this type of property in this local authority is £%{soft_min_for_period}."
-      max:
-        title_text: "You told us the rent is %{brent}"
-        hint_text: "The maximum rent expected for this type of property in this local authority is £%{soft_max_for_period}."
+      outside_range_title: "You told us the rent is %{brent}"
+      min_hint_text: "The minimum rent expected for this type of property in this local authority is £%{soft_min_for_period}."
+      max_hint_text: "The maximum rent expected for this type of property in this local authority is £%{soft_max_for_period}."
```

Comment:
I had removed these, but I reinstated them as they are still referenced in the 21/22 form. I don't have a great grip on what happens in terms of moving between different years.
I would slightly prefer these lines to no longer exist if they are no longer used. Alternative options:
- allow the 21/22 form to be broken because no one sees it any more?
- update the 21/22 form to the new copy
- remove these strings from the yml and just put them directly in the json, I assume I can't actually do this since they take variables which are retrieved on the step in between the json and the yml